app/serializers/obituary_serializer.rb

```diff
@@ -0,0 +1,19 @@
+class ObituarySerializer
+
+  def format_obituaries(obituaries)
+    obituaries.map do |obituary|
+      {
+        "data": {
+                  "type": "obituary",
+                  "attributes": {
+                                  'headline': obituary.headline,
+                                  'abstract': obituary.abstract,
+                                  'web_url': obituary.web_url,
+                                  'pub_date': obituary.pub_date,
+                                  'word_count': obituary.word_count
+                                }
+                }
+      }
```

Comment:
I wasn't able to get a serializer gem working so had to handroll this.

Reply:
Darn, would be interested in seeing if we can get a serializer to work in the future. Just for the practice! I volunteer as tribute.

Reply:
Nice, sounds good!

Reply:
awesome!

Reply:
![](https://media3.giphy.com/media/RO023EYTyk5yg/giphy.gif?cid=5a38a5a2mvxhtwlk6lbh5u2jx1lsyzwzpemz2l3d9yeuesyn&rid=giphy.gif)